content/docs/integrations/aws-resources/index.md

```diff
@@ -10,27 +10,80 @@ integrationType: Catalog data source
 
 ## Introduction
 
-Roadie has the capability to automatically ingest resources from AWS. This is done by automatic discovery and currently ingests all configured resource types per AWS region.
+Roadie has the capability to automatically ingest resources from AWS. This is done by automatic discovery and currently ingests all configured resource types per AWS region. You can configure AWS resource ingestion either by manually adding standalone account/region configurations or by using AWS organization structure to autodiscover accounts and their resources. The discovered AWS resources will be registered as `Resource` kind entities within Roadie. The type of the `Resource` entity indicates the AWS resource type.
 
 This guide describes how to set up Roadie to access your specific AWS resources and automatically ingest them.
 
 
-##  Step 1: Get the roadie IAM details
+## Configuring connectivity to AWS
+
+### Step 1: Get the roadie IAM details
 
 Navigate to `Administration > Settings > AWS Resources` and make a note of the Roadie backend role ARN and account ID. This is mentioned on the title text of the AWS Resources integration settings page.
 
-##  Step 2: Create a federated role in your account for Roadie
+### Step 2: Create a federated role in your account for Roadie
 
 Follow the steps [here](/docs/details/accessing-aws-resources) to create the role. 
 
 The role needs to follow this naming convention `arn:aws:iam::*:role/<your-tenant-name>-roadie-read-only-role` where <your-tenant-name> matches your organisation's name used in the url of your Roadie instance.
 
+> ⚠️ The enforced naming convention for acceptable assumable roles dictates that the role name needs to start with text `<tenant-name>-roadie-`. If other naming conventions are used, the role assumption is blocked by security measures.
+
 You'll need to attach policies to the role to be able to retrieve information about the resources you want ingested. The supported resources and their needed policies are listed in a table at the end of this page. You can use the same role for multiple resource types as long as the needed permissions are granted to it.
 
-For quick experimentation, you can use `AWS<ResourceType>ReadOnlyAccess` policies provided by AWS, but the best practice is to allow only specific needed operations.
+For quick experimentation, you can use `AWS<ResourceType>ReadOnlyAccess` policies provided by AWS, but the best practice is to allow only specific needed operations. You can see the needed policies in a table at the end of this page.
+
+### Step 3: Configure external id
+
+It is best practice (and mandatory for `autodiscovery` configuration) to configure an external id for your assumable role. 
+
+**(A)**
```

Comment:
What do you think about Option 1 and Option 2? Something like:
Option 1: Autodiscovery
~~~~
Option 2: Standalone (or could be Manual)